docs/api/api_docs.yaml

```diff
@@ -999,17 +1003,28 @@ paths:
       description: |-
         This request retrieves a file attachment associated with the document. 
 
-        The raw data of the associated attachment is returned (just as if you were accessing a static file). The `Content-Type` response header is the same content type set when the document attachment was added to the database. The `Content-Disposition` response header will be set instead if the content type is vulnerable to a phishing attack, causing the browser to download the file instead of display it (unless overriden by by database config option `serve_insecure_attachment_types`).
+        The raw data of the associated attachment is returned (just as if you were accessing a static file). The `Content-Type` response header is the same content type set when the document attachment was added to the database. The `Content-Disposition` response header will be set if the content type is considered unsafe to display in a browser (unless overriden by by database config option `serve_insecure_attachment_types`) which will force the attachment to be downloaded.
 
         If the `meta` query parameter is set then the response will be in JSON with the additional metadata tags.
       parameters:
         - $ref: '#/components/parameters/rev'
-        - schema:
+        - name: content_encoding
+          schema:
+            type: boolean
+            default: 'true'
+          in: query
+          description: 'Set to false to disable the `Content-Encoding` response header.'
+        - name: Range
+          schema:
+            type: string
+          in: header
+          description: 'RFC-2616 bytes range header. For example, bytes=123-456.'
```

Comment:
Again my bad w/ suggestion - I've seen the correct way of specifying an example:

```suggestion
          description: 'RFC-2616 bytes range header.'
          example: 'bytes=123-456'
```